dev/io.openliberty.microprofile.reactive.messaging30.internal_fat_tck/fat/src/io/openliberty/microprofile/reactive/messaging30/tck/ReactiveMessagingTCKLauncher.java

```diff
@@ -53,7 +54,10 @@ public static void tearDown() throws Exception {
 
     @Test
     @Mode(TestMode.FULL)
-    @AllowedFFDC({ "org.jboss.weld.exceptions.DeploymentException", "com.ibm.ws.container.service.state.StateChangeException", "jakarta.enterprise.inject.spi.DeploymentException" }) // The tested deployment exceptions cause FFDC so we have to allow for this.
+    // Allowing InstanceNotFoundException as it is possible for mpmetrics to be queried during server shutdown when
```

Comment:
Can you combine this comment with the one that is at the end of the `AllowedFFDC` line as we shouldn't really have comments about the same line being both above and below it